tools.yml

```diff
@@ -1133,19 +1088,24 @@ tools:
     mem: 12
     env:
       _JAVA_OPTIONS: -Xmx{int(mem)}G -Xms1G
+  toolshed.g2.bx.psu.edu/repos/iuc/extract_genomic_dna/Extract genomic DNA 1:
```

Comment:
Nice to know that you can use spaces like this in tool names :)

Reply:
Yeah it works although it's not something we *should* be doing. This is a veeeeeeeeeery old tool.